mule-migration-tool-api/src/main/java/com/mulesoft/tools/migration/project/model/pom/Parent.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020, Mulesoft, LLC. All rights reserved.
+ * Use of this source code is governed by a BSD 3-Clause License
+ * license that can be found in the LICENSE.txt file.
+ */
+package com.mulesoft.tools.migration.project.model.pom;
+
+/**
+ * Represents a Parent in the pom model. By default its type is jar.
+ * 
+ * @author Mulesoft Inc.
+ * @since 1.0.0
+ */
+public class Parent {
+
+  private final org.apache.maven.model.Parent parent;
+
+  protected Parent(org.apache.maven.model.Parent parent) {
+    this.parent = parent;
```

Comment:
Add validation for a null value parent. You can use guava preconditions:
`Preconditions.checkArgument(parent != null,"The parent cannot be null");`

Reply:
the constructor is protected and there is only one caller. Not needed